client/app/bundles/ConifyClient/startup/App.jsx

```diff
@@ -10,7 +10,7 @@ import HelloWorld from '../containers/HelloWorld';
 // This code here binds your smart component to the redux store.
 // railsContext provides contextual information especially useful for server rendering, such as
 // knowing the locale. See the React on Rails documentation for more info on the railsContext
-const App = (props, _railsContext) => {
+const App = (props/*, _railsContext*/) => {
```

Comment:
Linter warning for unused variable. That'ts why I inline commented it, it indicates it could still be used but not by the current code.